.nvmrc

```diff
@@ -0,0 +1 @@
+v14.15.5
```

Comment:
Let's remove it from here, I don't think we should have such user-specific config files commited to repo

Reply:
No problem. I will remove it.